docs/products/kafka/kafka-connect/howto/gcp-bigquery-sink.rst

```diff
@@ -0,0 +1,245 @@
+Create a Google BigQuery sink connector
+=======================================
+
+The Google BigQuery sink connector enables you to move data from an Aiven for Apache Kafka® cluster to a set of Google BigQuery tables for further processing and analysis. 
+
+
+.. _connect_bigquery_sink_prereq:
+
+Prerequisites
+-------------
+
+To setup an Google BigQuery sink connector, you need an Aiven for Apache Kafka service :doc:`with Kafka Connect enabled <enable-connect>` or a :ref:`dedicated Aiven for Apache Kafka Connect cluster <apache_kafka_connect_dedicated_cluster>`. 
+
+Furthermore you need to follow the steps :doc:`to prepare the GCP account <gcp-bigquery-sink-prereq>` and collect the following information about the target BigQuery upfront:
+
+* ``GCP_PROJECT_NAME``: The GCP project name where the target Google BigQuery is located
+* ``GCP_SERVICE_KEY``: A valid GCP service account key for the ``GCP_PROJECT_NAME``. To create the project key review the :ref:`dedicated document <gcp-bigquery-sink-connector-google-account>`
+
+.. Warning::
+
+    The GCP BigQuery sink connector accepts the ``GCP_SERVICE_KEY`` JSON service key as string, therefore all  ``"`` symbols within it must be escaped ``\"``.
+
+    The ``GCP_SERVICE_KEY`` parameter should be in the format ``{\"type\": \"service_account\",\"project_id\": \"XXXXXX\", ...}``
+
+    Additionally, any ``\n`` symbols contained in the ``private_key`` field need to be escaped (by substituting with ``\\n``)
+
+* ``BIGQUERY_DATASET_NAME``: The BigQuery dataset name, as defined in the :ref:`dedicated pre-requisite step <gcp-bigquery-sink-connector-bigquery-dataset>`
+* ``TOPIC_LIST``: The list of topics to sink divided by comma
+* ``APACHE_KAFKA_HOST``: The hostname of the Apache Kafka service, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PORT``: The Apache Kafka's schema registry port, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_USER``: The Apache Kafka's schema registry username, only needed when using Avro as data format
+* ``SCHEMA_REGISTRY_PASSWORD``: The Apache Kafka's schema registry user password, only needed when using Avro as data format
+
+
+.. Note::
+
+    The ``SCHEMA_REGISTRY`` related parameters are available in the Aiven for Apache Kafka® service page, *Overview* tab, and *Schema Registry* subtab
```

Comment:
Might be worth including options where schema registry is not needed (for example, when a user has an existing table defined)
` "value.converter.schemas.enable": false,
 "key.converter.schemas.enable": false,`

Reply:
I believe we need to come up with an example on this, rather than just showing the values. Will think on it